src/main/kotlin/racingcar/RacerName.kt

```diff
@@ -0,0 +1,50 @@
+package racingcar
+
+class RacerName(private val racerCrew: String = "") {
+    fun verifyRacerName(): MutableList<String> {
+        val getVerifyRacerNames: MutableList<String> = blendRacerName()
+        exceptionRacerName(getVerifyRacerNames)
+        return getVerifyRacerNames
+    }
+
+    private fun blendRacerName(): MutableList<String> {
+        val personalRacerName = racerCrew.split(",").map { it.trim() }
+        return personalRacerName.toMutableList()
+    }
+
+    private fun exceptionRacerName(racerNames: MutableList<String>) {
+        checkRacerNameLength(racerNames)
+        checkRacerNameBlank(racerNames)
+        checkRacerTotalRange(racerNames)
+    }
+
+    private fun checkRacerNameLength(getVerifyRacerNames: MutableList<String>): Boolean {
+        for (name in getVerifyRacerNames) {
+            compareNameLength(name)
+        }
+        return true
+    }
+
+    private fun checkRacerNameBlank(getVerifyRacerNames: MutableList<String>): Boolean {
+        for (name in getVerifyRacerNames) {
+            existNameBlank(name)
+        }
+        return true
+    }
+
+    private fun checkRacerTotalRange(getVerifyRacerNames: MutableList<String>): Boolean {
+        if (getVerifyRacerNames.size !in RACER_MINIMUM..RACER_MAXIMUM)
+            throw IllegalArgumentException(Message.RACER_NAME_RANGE_ERROR.messageText)
+        return true
+    }
```

Comment:
if와 !in 연산자 대신 [require()](https://seosh817.tistory.com/155)을 사용하면 가독성이 더 좋을 것 같습니다.
```suggestion
    private fun checkRacerTotalRange(getVerifyRacerNames: MutableList<String>): Boolean {
        require(getVerifyRacerNames.size in RACER_MINIMUM..RACER_MAXIMUM) {
            Message.RACER_NAME_RANGE_ERROR.messageText
        }
        return true
    }
```

Reply:
다음 미션의 예외 처리 기능 구현에서 require()를 사용해 가독성을 높여보겠습니다. 의견 감사합니다!